scripts/toolboxSrc/postDeploy.ts

```diff
@@ -1,13 +1,34 @@
 import fs from 'fs'
 import path from 'path'
 import getRethink from '../../packages/server/database/rethinkDriver'
-import db from '../../packages/server/db'
 import getProjectRoot from '../webpack/utils/getProjectRoot'
+import getRedis from '../../packages/server/utils/getRedis'
 
 const PROJECT_ROOT = getProjectRoot()
 
 const flushSocketConnections = async () => {
-  return db.writeTable('User', {connectedSockets: []})
+  const redis = getRedis()
+  const userPresenceStream = redis.scanStream({match: 'presence:*'})
+  userPresenceStream.on('data', async (keys) => {
+    if (!keys?.length) return
+    userPresenceStream.pause()
+    const writes = keys.map((key) => {
+      return ['del', key]
+    })
+    await redis.multi(writes).exec()
```

Comment:
lets get rid of the await as well as the pause/resume. we'll know it finished up when the stream `end` fires.